### PR TITLE
[Ready] More cakes and pies, as well as resprites a few others

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Adds in more tarts - pies
Mime Tart, Lava Coco flow Tart, berry tart 
Adds in a few more cakes
Vanilla cake - real this time
Clown cake - Honk!
New book only get able via Dinner-o-mat that teaches you how to makes these

## Why It's Good For The Game

More colorful foods and crafting as well as using the locked crafting system - Maybe better looking cakes

image

## Changelog
:cl:
add: Cakes/Tarts/book
imageadd: new cake/tarts/books sprites
/:cl:

### CODE REVIEW

**Comment by Angustmeta** [comment]
Which old ones were resprited?

**Comment by FantasticFwoosh** [comment]
Hmm interesting, i wonder if this could be a move foward towards countering crafting feature-bloat.

I can think of a few good locations for things like this in maintenance to replace the HTML tutorial books.

**Comment by zombiegums** [comment]
The clown cake makes me happy.

**Comment by Angustmeta** [comment]
> Checks then Icon different it should show you the old - > modifed

I can't into git and the good git guides are hidden deep within mountains of bad guides or other stuff I can't understand. Googling "git icon diff" or "github image diff" finds me nothing. Can you rephrase please? I need it for wiki.

**Comment by Angustmeta** [comment]
Very nice, thanks

**Comment by Quartich** [comment]
For screenshots I would recommend nearest neighbor and 64x to make it look better, looks a bit funky in your's
